src/pages/waiting-room/cat-be/waiting-room.cat-be.page.ts

```diff
@@ -178,6 +192,12 @@ export class WaitingRoomCatBEPage extends BasePageComponent implements OnInit {
     }
   }
 
+  isJournalDataInvalid = (journalData: CatBEUniqueTypes.JournalData): boolean => {
+    return isEmpty(journalData.examiner.staffNumber)
+      || isEmpty(journalData.candidate.candidateName)
+      && isEmpty(journalData.candidate.driverNumber);
+  }
+
```

Comment:
same comment as before - would like to see brackets to clarify the logic.  After looking at the tests I can see it would be isEmpty(....) || ( isEmpty() && isEmpty())

Reply:
Okay, I can add them, but by default the JS interpreter will execute the `&&` operation first and then the `||`

Reply:
Indeed - but I always prefer explicit bracketing - then there's no wondering if the person writing the code took account of operator precedence, or no issues with people reading the code not having the operator precedence memorised.

Reply:
Okay, so here is an alternative. I did format the text differently so that it's more obvious the execution order